source/operations/install-deploy-manage/multi-site-replication.rst

```diff
@@ -453,13 +453,17 @@ The new site must meet the following requirements:
       #. Add site replication configuration
```

Comment:
👆🏻 line 430 (in existing text) has punctuation when its sibling list items do not. Good time to standardize.